controller/argument_value_resolver.rst

```diff
@@ -32,8 +32,8 @@ Symfony ships with the following value resolvers in the
     works like :doc:`autowiring </service_container/autowiring>`.
 
 :class:`Symfony\\Component\\HttpKernel\\Controller\\ArgumentResolver\\SessionValueResolver`
-    Injects the configured session class extending ``SessionInterface`` if
-    type-hinted with ``SessionInterface`` or a class extending
+    Injects the configured session class implementing ``SessionInterface`` if
+    type-hinted with ``SessionInterface`` or a class implementing
     ``SessionInterface``.
```

Comment:
@javiereguiluz Do you have an idea what the second part of the sentence (starting with "or") is supposed to express? I do not really see how this add more information.